src/test/java/org/openelisglobal/patient/PatientServiceTest.java

```diff
@@ -38,36 +52,45 @@ public void tearDown() {
 
     @Test
     public void createPatient_shouldCreateNewPatient() throws Exception {
-        String firstName = "John";
-        String lastname = "Doe";
-        String dob = "12/12/1992";
-        String gender = "M";
-        Patient pat = createPatient(firstName, lastname, dob, gender);
-
-        Assert.assertEquals(0, patientService.getAllPatients().size());
-        // save patient to the DB
-        String patientId = patientService.insert(pat);
```

Comment:
This  function is about creating a patient. 
Meaning we would have a patient created with in this fucntion

Reply:
And also you need to assert the number of patients before and after